dao/mongodb.rya/src/main/java/mvm/rya/mongodb/dao/SimpleMongoDBStorageStrategy.java

```diff
@@ -8,9 +8,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
```

Comment:
For future reference, it would be a lot easier to do your reviews if you would doing the whitespace and style changes as a separate commit from the logic changes.